pkgs/applications/video/obs-studio/default.nix

```diff
@@ -46,19 +51,20 @@ let
 in
 mkDerivation rec {
   pname = "obs-studio";
-  version = "27.2.4";
+  version = "28.0.3";
 
   src = fetchFromGitHub {
     owner = "obsproject";
     repo = "obs-studio";
     rev = version;
-    sha256 = "sha256-OiSejQovSmhItrnrQlcVp9PCDRgAhuxTinSpXbH8bo0=";
+    sha256 = "sha256-+4H1BjEgxqkAEvRyr2Tg3wXutnMvlYQEdT5jz644fMA=";
     fetchSubmodules = true;
   };
 
   patches = [
     # Lets obs-browser build against CEF 90.1.0+
```

Comment:
Do we still need this?

Reply:
The code-part patched is consistent since 3 years, obs still builds without but I think there was a need for it even I can explain/have a scenario to test it. How should we handle this?

Reply:
I don't see any obvious incompatibilities and it's better security. If there's no breakage observed I think it's better to keep it.